DigitalLearningSolutions.Web/Views/NotificationPreferences/Index.cshtml

```diff
@@ -34,6 +52,25 @@
           </span>
         </summary>
         <div class="nhsuk-details__text">
+          <dl class="nhsuk-summary-list preference-list">
+            @foreach (var notification in Model.DelegateNotifications)

```

Comment:
This code is duplicated and fixing both was a bit more fiddly than I'd like. Before I merge this, is there a good example of how to make it a component I can call in both places?

Reply:
Yes, could you use partial views using a Partial Tag Helper - https://docs.microsoft.com/en-us/aspnet/core/mvc/views/partial?view=aspnetcore-3.1

There is an example how one is used in LearningPortal/Current index view - the cards are rendered as `<partial name="SelfAssessments/_SelfAssessmentCard" model="currentLearningItem" />`